ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -84,11 +84,18 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
     transition,
```

Comment:
non-blocking question: should the sessions route be updated  as well or will that be in a separate PR ?

Reply:
That is a good point. I will add it to sessions now!

Reply:
Interesting. I added this same logic to the sessions route. However, when you change the scope at the top of the app, it redirects you to the targets route.

Reply:
Updated!